src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/view-index-top.jelly

```diff
@@ -66,4 +62,8 @@ THE SOFTWARE.
       </div>
     </j:otherwise>
   </j:choose>
+  <!-- give actions a chance to contribute summary item -->
```

Comment:
I proposed to move the button up. As an example, with the Ownership plugin the current layout is showing this:

![before](https://user-images.githubusercontent.com/6858329/73331590-fed05e80-42af-11ea-92a2-0f5800b86337.png)

By changing the order, we get something like this:

![after](https://user-images.githubusercontent.com/6858329/73331645-30492a00-42b0-11ea-9307-3f23e25ae15b.png)

More appealing and also closer to what Jobs are showing. Let me know what you think. Maybe this can be improve even more.